boot/native_image_test.go

```diff
@@ -116,24 +116,4 @@ func testNativeImage(t *testing.T, context spec.G, it spec.S) {
 			Expect(layer.LayerTypes.Launch).To(BeFalse())
 		})
 	})
-	context("Boot @argfile is found", func() {
```

Comment:
As above, since this tests the same functionality

Reply:
re instated!